pkg-linux/libs3.spec

```diff
@@ -0,0 +1,81 @@
+Summary: C Library and Tools for Amazon S3 Access
```

Comment:
Instead of copying - symlink or change mock_build to look in .. and add the version/release defaults/params to the main spec.